### ISSUE TITLE
Cannot set *Message Type* when using `protobuf`

### ISSUE THREAD

**Issue by ihomway** [ProxymanApp/Proxyman#706]
### Proxyman version?
**Version 2.13.0 (21300)**

### macOS Version?
**Version 11.0.1 (20B29)**

### Steps to reproduce 
I have a photo file like this:
```protobuf
syntax = "proto3";

package example.api.feeds.v2
import "google/protobuf/any.proto";
option go_package = "git.example.com/api/feeds/v2;v2";
option java_multiple_files = true;
option java_package = "com.exaple.api.feeds.v2";
option objc_class_prefix = "FeedsV2";

service Feed {
    rpc list(Req) returns (Reply);
}

message Req {
    int32 page = 1;
}

message Reply {
    repeated ListItem items = 1;
}

message ListItem {
    string content = 1;
}
```

And then I'm trying to parse the response which is encoded as protobuf. Here is the settings:

1. I add the photo file:
  ![Screen Shot 2020-11-25 at 11 56 06 AM](https://user-images.githubusercontent.com/9472797/100181450-38e0b000-2f15-11eb-82b8-c3662b76efa4.png)

2. I cannot set the message type:
![Screen Shot 2020-11-25 at 11 58 49 AM](https://user-images.githubusercontent.com/9472797/100181675-c91ef500-2f15-11eb-9ff8-c06568e53656.png)

All I can find in the drop-down list of **Message Type** are like "google.protobuf.*".

### Expected behavior
I can find the message type correctly.

**Comment by NghiaTranUIT** [comment]
Hi @ihomway 

Look like Proxyman Protobuf couldn't parse your proto file properly since it is generated by gRPC protobuf.

If you import this file (add ; in syntax line, remove import line)

```
syntax = "proto3";

package example.api.feeds.v2;

service Feed {
    rpc list(Req) returns (Reply);
}

message Req {
    int32 page = 1;
}

message Reply {
    repeated ListItem items = 1;
}

message ListItem {
    string content = 1;
}
```
You can select the package, but I'm not sure if you're able to see the gRPC content. Let give it a try.

<img width="839" alt="Screen Shot 2020-11-25 at 11 17 26" src="https://user-images.githubusercontent.com/5878421/100182838-21eb8f00-2f10-11eb-956d-6992e32abec5.png">

**Comment by NghiaTranUIT** [comment]
Alternatively, there is a easy way to see gRPC traffic by using Atlantis. You can follow this doc: https://github.com/ProxymanApp/atlantis#2-my-app-use-grpc

By using Atlantis, you don't need to add Protobuf file anymore. but you have to add extra code to initial the request/response and sending to Atlantis. Finally, you can see the traffic on Proxyman as usual 👍

**Comment by ihomway** [comment]
Thank you for your help, I will try Atlantis and will let you if it works or not.

**Comment by NghiaTranUIT** [comment]
Yes, please let me know both scenario. Basically, Proxyman might only supports "normal" Protobuf message, not gRPC protobuf

**Comment by ben-xD** [comment]
I have the same issue, and i am not using gRPC protobuf. Unfortunately, my project is quite complicated with lots of imports. I tried adding the specific protobuf which contains the message I want to decode. I also tried removing the import lines from this file, but still, same issue, the protobuf message I want is not available in drop down menu.

It is also quite tedious to add all the files manually to Protobuf Schema List. This because A.proto might import B.proto, which might import C.proto. It would be great if I can just select the directory which contains all the protobufs I use.

**Comment by NghiaTranUIT** [comment]
@ben-xD it might have the issue when parsing Protobuf that imports different Protobuf files.

For simple solution, can you use Atlantis to inspect the traffic 🤔 Tutorial here: https://github.com/ProxymanApp/atlantis#2-my-app-use-grpc

One of drawback is that we could not use any debugging tools.
